types/hooks/screens/useChatScreenHook.types.ts

```diff
@@ -13,6 +19,10 @@ interface UseChatScreenHook {
   selfAnonUserInfo: UserSchema | null;
   handleUserName: (item: ChatSchema) => string;
   updateChatContinuity: (chatsData: ChatSchema[]) => ChatSchema[];
+  setIsLoadingFetchAllMessage: React.Dispatch<React.SetStateAction<boolean>>;
+  refetchMessage: <TPageData>(
+    options?: (RefetchOptions & RefetchQueryFilters<TPageData>) | undefined
+  ) => Promise<QueryObserverResult<BaseDbSchema[], unknown>>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setIsLoadingFetchAllMessage` and `refetchMessage` functions have been added to the `UseChatScreenHook` interface. However, it's not clear from this hunk whether these functions are being used correctly in the context of the application. It would be beneficial to see how these functions are implemented and used in the component that uses this hook. If they are used correctly, then these changes look good.


<!-- This is an auto-generated comment by OSS CodeRabbit -->